### PR TITLE
add hullcss blog feed to feeds.json

### PR DESCRIPTION
**Pull request by CrimsonTome**
This PR adds [HullCSS](https://hullcss.org/)' blog to the feed list

### CODE REVIEW

**Review by sbrl — APPROVED** [review]
Looks good! I assume though that the author name will change every year?

**Comment by CrimsonTome** [comment]
> Looks good! I assume though that the author name will change every year?

amended

I will un-draft this PR when all exec members have written their introduction :smile:

**Comment by CrimsonTome** [comment]
resolved merge conflicts, PR is ready to merge

File: feeds.json [hunk 1]
```diff
@@ -29,6 +29,15 @@
 		"github_username": "KieranRobson",
 		"feed_uri": "https://docs.kieranrobson.com/feed.xml"
 	},
+		{
+		"author_name": "Kieran Robson",
+		"github_username": "KieranRobson",
+		"feed_uri": "https://blog.kieranrobson.com/feed.xml"
+	},
```
Comment (sbrl) [inline feeds.json:36]: Erm isn't Kieran already listed in this file?

**Pull request merged**